scripts/generate_asm.md

```diff
@@ -0,0 +1,73 @@
+This is a python script for generating arbitrary asm files.
```

Comment:
Let's rename this file to README.md for now, so GitHub picks it up automatically.